src/pages/teamHome/teamJoin.tsx

```diff
@@ -1,346 +1,387 @@
+        const temp: any[] = [];
+        collection.forEach((doc: any) => {
+          temp.push(doc);
+        });
+        getData(temp);
+      }
+    );
+    return () => {
+      console.log('unsubscribing from team join page');
+      unsubscribe();
+    };
+  }, [ctx.user, ctx.team]);
+
+  // move to the team creation page
+  const moveToCreateTeam = () => {
+    history.push('/app/teamcreation');
+  };
+
+  // check team deadline
+  function displayPage() {
+    if (buttonValid === false) {
+      return <>{DisplayTeams(allTeams)}</>;
+    } else {
+      return (
+        <h1>The deadline to join or create a team was {adData.teamDate}.</h1>
+      );
+    }
+  }
+
+  return (
+    <IonPage>
+      <IonHeader>
+        <NavBar>
+          <IonTitle> Team Join </IonTitle>
+        </NavBar>
+      </IonHeader>
+      <IonContent fullscreen className="team-join">
+        <IonRow>
+          <IonCol sizeXs="12" sizeMd="6">
+            <IonCard className="join-create-team-card">
+              <IonCardHeader
+                style={{ display: 'flex', justifyContent: 'center' }}
+              >
+                <img alt="Picture of walktober team" src={team} />
+              </IonCardHeader>
+              <IonCardTitle class="ion-text-center">
+                Join an Existing Team!
+              </IonCardTitle>
+              <IonCardSubtitle class="ion-text-center">
+                Enter the team name, password (if private) and click Join Team!
+              </IonCardSubtitle>
+              <IonCardContent>
+                <IonRow>
+                  <IonCol sizeXs="6" sizeMd="4">
+                    <IonItem>
+                      <IonLabel position="floating">Team</IonLabel>
+                      <IonInput
+                        onIonChange={(e) => setJoin(e.target.value as string)}
+                      ></IonInput>
+                    </IonItem>
+                  </IonCol>
+                  <IonCol sizeXs="6" sizeMd="4">
+                    <IonItem>
+                      <IonLabel position="floating">Team Password</IonLabel>
+                      <IonInput
+                        name="password"
+                        onIonChange={(e) => setPass(e.target.value as string)}
+                      ></IonInput>
+                    </IonItem>
+                  </IonCol>
+                  <IonCol className="join-create-button" sizeXs="12" sizeMd="4">
+                    <IonButton disabled={buttonValid} onClick={toJoin}>
+                      Join Team
+                    </IonButton>
+                  </IonCol>
+                </IonRow>
+              </IonCardContent>
+            </IonCard>
+          </IonCol>
+
+          <IonCol sizeXs="12" sizeMd="6">
+            <IonCard className="join-create-team-card">
+              <IonCardHeader
+                style={{ display: 'flex', justifyContent: 'center' }}
+              >
+                <img alt="picture of team leader" src={solo} />

```

Comment:
Can we please change this to "Art depicting a team leader raising her fist in triumph"

Reply:
done.